include/layers/InputLayer.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <stdexcept>
+#include <string>
+
+#include "layers/Layer.hpp"
+
+class InputLayer : public Layer {
+ public:
+  InputLayer() = default;
```

Comment:
Input layer accepts parameters like:
- layout (NCHW, NHWC)
- coefficients for image normalization (mean, std, ...)

Reply:
And which one does we receive at the entrance of NCHW or NHWC? Or is it necessary to consider both options?

Reply:
fixed